test/run.sh

```diff
@@ -34,10 +34,12 @@ function runCommandAndCompareToSnapshot {
       exit 1
     fi
 
-    eval "$LOCAL_COMMAND$AUTH --FOR-TESTS $ARGS" 2>&1 \
+    (eval "$LOCAL_COMMAND$AUTH --FOR-TESTS $ARGS" || true) 2>&1 \
```

Comment:
All of these `|| true`s tell bash that the command might fail. They should be here either way, it's a correctness issue, but pipefail enforces it.
The subshell makes it so that it's not the `true` that gets redirected to `/dev/null/`, but the whole thing.